services/db/AxiosInstance.ts

```diff
@@ -0,0 +1,42 @@
+import { AxiosRequestConfig, AxiosResponse } from "axios"
```

Comment:
could we put this in `services/api` as this isn't directly related to the db

Reply:
Good point, https://github.com/isomerpages/isomercms-backend/pull/440/commits/3148628a401ea4f4675d5729d350de8002d3b4c1